docs/docs/manual/grelfunctions.md

```diff
@@ -377,17 +377,10 @@ Returns the array with duplicates removed. Case-sensitive. For example, `[ "al",
 As of OpenRefine 3.4.1, uniques() reorders the array items it returns; in 3.4 beta 644 and onwards, it preserves the original order (in this case, [ "al", "Joe", "Bob", "Al" ]). 
 
 ## Date functions
-If you have a column full of dates in a more common format (such as yyyy/mm/dd or YYYYMMDD) you can convert these using [“Transform” > “Common transforms …” > “to date”](cellediting#data-type-transforms), or the GREL function toDate(). You may wish to create a new column with the transformation, in order to also preserve the more human-readable version.
-
-OpenRefine uses [Date.parse()](https://www.w3schools.com/jsref/jsref_parse.asp) to recognize a variety of formats and convert them, including converting from other time zones to UTC:
-
-![A screenshot of different date formats being converted, and one error.](/img/dates.png)
-
-You may need to do some reformatting if your dates are not being recognized by the toDate() function. For example, in the image above, the date that includes “7AM” is giving an error message, but the ones with “2:42 PM” and “3:22PM” are being converted.
 
 ###### now()
 
-Returns the current time in the [ISO 8601 extended format](exploring#data-types). For example, 12:58pm on October 28th 2020 returns [date 2020-10-28T16:58:32Z].
+Returns the current time according to your system clock, in the [ISO 8601 extended format](exploring#data-types) (converted to UTC). For example, 12:58pm on October 28th 2020 returns [date 2020-10-28T16:58:32Z].
```

Comment:
Sorry - a further refinement. That example only works if you specify the timezone of the original date/time

Reply:
I mean - for me at 12:58pm (and 32 seconds) on October 28th 2020 `now()` would return `[date 2020-10-28T12:58:00Z]`
Also - probably should include seconds in the text or have the returned time ending `:00`

Reply:
Of course, I should've formatted that right. Thanks.